packaging/linux/deb/debroot.go

```diff
@@ -147,8 +146,8 @@ func Debroot(ctx context.Context, sOpt dalec.SourceOpts, spec *dalec.Spec, worke
 				return in
```

Comment:
This whole block needs to be removed.
(I'd do it as a suggestion but GH won't let me select the starting from L145...).

This along with removing the `fix-sources` stuff in the rules file fixes the issue we discussed.